app1React/src/root.component.js

```diff
@@ -6,20 +6,15 @@ import reactLogo from '../assets/react-logo.png'
 
 export default class Root extends React.Component {
 
-    state = {store: null, globalEventDistributor: null};
+    state = {
+      store: this.props.customProps.store,
```

Comment:
We can just get the store and event distributor off of the props, instead of calling setState later with them.